corehq/motech/repeaters/expression/repeater_generators.py

```diff
@@ -39,3 +39,29 @@ def get_url(self, repeat_record, url_template, payload_doc):
                 for template_var in required_template_vars
             }
         )
+
+
+class ArcGISFormExpressionPayloadGenerator(ExpressionPayloadGenerator):
+
+    def get_url(self, repeat_record, url_template, payload_doc):
+        if not (
+            toggles.UCR_EXPRESSION_REGISTRY.enabled(repeat_record.domain)
+            and toggles.ARCGIS_INTEGRATION.enabled(repeat_record.domain)
+        ):
+            return ""
+
+    @property
+    def content_type(self):
+        return 'application/x-www-form-urlencoded'
```

Comment:
I find it slightly weird that they're expecting urlencodedd content which is really json, or am I misinterpreting the formatted_payload?

(just curious about how the integration works, not a comment on your code)

Reply:
Yeah this seems to be the expected content type. Omitting this causes the payload to submit in default `application/json`, however when testing with JSON format the API responded that the params were all missing which indicates that they don't very much like that format.

Looking at their [Postman example](https://www.postman.com/esridevs/workspace/arcgis-location-services/request/15786767-8b777725-387b-402f-9033-4565a84111c4?tab=overview) of a POST request, they have also formatted all the params to be in `x-www-form-urlencoded` format.

Reply:
@Charl1996 yes, confirming your curiosity. The way ArcGIS does this is definitely weird. :roll_eyes: